package-lock.json

```diff
@@ -0,0 +1,11 @@
+{
+  "requires": true,
+  "lockfileVersion": 1,
+  "dependencies": {
+    "markdown-toolbar-element": {
+      "version": "0.2.0",
+      "resolved": "https://registry.npmjs.org/markdown-toolbar-element/-/markdown-toolbar-element-0.2.0.tgz",
```

Comment:
I wonder if it's the missing hash at the end of this url that's causing a problem?

https://github.com/alphagov/content-publisher/blob/1c6209ee2f11a3ae4c348bd8d7489632451fa1b6/yarn.lock#L832

Reply:
That's the only difference I could find between the setup here an in content-publisher.